htdocs/src/AppBundle/Entity/PageGroup.php

```diff
@@ -0,0 +1,257 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+use Doctrine\ORM\Mapping\OneToMany;
+
+/**
+ * PageGroup
+ *
+ * @ORM\Table(name="page_groups")
+ * @ORM\Entity(repositoryClass="AppBundle\Repository\PageRepository")
+ */
+class PageGroup
+{
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="id", type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="slug", type="string", length=255, unique=true)
+     */
+    private $slug;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="meta_keywords", type="string", length=255, nullable=true)
+     */
+    private $metaKeywords;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="meta_description", type="string", length=255, nullable=true)
+     */
+    private $metaDescription;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="meta_social", type="text", nullable=true)
+     */
+    private $metaSocial;
+
+    /**
+     * @var \DateTime
+     *
+     * @ORM\Column(name="last_changed", type="datetime")
+     */
+    private $lastChanged;
+
+    /**
+     * @var bool
+     *
+     * @ORM\Column(name="active", type="boolean")
+     */
+    private $active;
```

Comment:
provide a default value true or false, else creation of new objects are invalid as this field is not declared as **nullable**